pinot-query-runtime/src/test/resources/queries/TableExpressions.json

```diff
@@ -0,0 +1,93 @@
+{
+  "where_clause_tests": {
+    "psql": "7.2.2",
+    "tables": {
+      "tbl": {
+        "schema": [
+          {"name": "strCol", "type": "STRING"},
+          {"name": "intCol", "type": "INT"}
+        ],
+        "inputs": [
+          ["foo", 1],
+          ["bar", 2],
+          ["alice", 42],
+          ["bob", 196883]
+        ]
+      }
+    },
+    "queries": [
+      { "sql": "SELECT * FROM {tbl} WHERE intCol > 5" },
+      { "sql": "SELECT * FROM {tbl} WHERE strCol IN ('foo', 'bar')" },
+      { "sql": "SELECT * FROM {tbl} WHERE intCol IN (196883, 42)" },
+      { "sql": "SELECT * FROM {tbl} WHERE strCol IN (SELECT strCol FROM {tbl} WHERE intCol > 100)" },
```

Comment:
Can we also add query (or a TODO) where the IN clause subquery runs a JOIN ?

Something like ....

```
SELECT C1, C2
FROM T1
WHERE C1 IN (
              SELECT T2.C1
              FROM T3
              INNER JOIN T2 ON T2.C3 = T3.C3
              WHERE .... 
           )
```

Reply:
added